astropy/tests/runner.py

```diff
@@ -48,6 +49,11 @@ def keyword(*args, **kwargs):
         return keyword
 
 
+@deprecated(
+    "6.1",
+    message="""Use pytest directly instead. You can read more about the motivation
+                    for the deprecation at https://github.com/astropy/astropy/issues/16177""",
```

Comment:
I'm surprised this survives formatting. I suspect the leading white space on the second line isn't trimmed at all.

Reply:
pre-commit messed this up in the first place! Fixed it in a new push.